### ISSUE TITLE
Feed doesn't contain Retweats

### ISSUE THREAD

**Issue by verneleem** [manikandanraji/twitter-clone-backend#7]
@manikandanraji reverse engineering your backend and the GraphQL server is appears that the feed query does not contain retweets only tweets. Is this correct, or did I miss something?

**Comment by manikandanraji** [comment]
You are correct. I am lazy and didn't queried the retweets when compiling the user feed.
